### PR TITLE
Revisit styles for display playback label for IE 11 compatability

### PR DESCRIPTION
**Pull request by boyntoni**
### This PR will...

Enforce consistency for the positioning of the playback label across browsers. This PR removes Flexbox for the labels' parent container, which was the cause of incorrect positioning in Internet Explorer.

As an alternative, setting the width of the ```.jw-idle-icon-text``` class to 100% achieves the desired result. In conjunction with the previously defined ```text-align: center``` rule, forcing the label to occupy width of the parent container, the label is centered across all browsers.

### Why is this Pull Request needed?

In IE, the "Play" playback label is not centered under the play icon; instead, it is aligned to the right.

### Are there any points in the code the reviewer needs to double check?

N/A

### Are there any Pull Requests open in other repos which need to be merged with this?

No

#### Addresses Issue(s):

JW8-2360

### CODE REVIEW

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🛠 There are modified src files, but no test changes. Add tests if you're able to.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  🗿 Set a milestone. It should be the ticket's fix version in JIRA.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by karimMourra** [comment]
@boyntoni please update your PR description, it is too vague. "Address the alignment issue in IE 11 with the display playback label." is more fitting as an answer to *Why the PR is needed*.
 *This PR will...* should provide a technical explanation of what you are changing. You are removing some CSS but you should describe why we do not need this CSS. I want to know why removing this CSS does not break anything, and why it was causing the misalignment.

**Comment by boyntoni** [comment]
@robwalch Will update the page and add playback variant to it.

@karimMourra  Thanks for the feedback, have updated description. Let me know if it needs further tweaking.

File: src/css/controls/imports/display.less [hunk 1]
```diff
@@ -55,9 +55,6 @@ display icons
     .jw-icon {
         .square(75px);
         cursor: pointer;
-        display: flex;
-        justify-content: center;
-        align-items: center;
```
Comment (robwalch) [inline src/css/controls/imports/display.less]: Why was this removed?

**Comment by robwalch** [comment]
@boyntoni I'm rerunning the build to look at the new view page variant.

**Comment by boyntoni** [comment]
@robwalch  Thanks. Will check it out, as well.
